src/main/java/frc/robot/Util.java

```diff
@@ -19,18 +18,24 @@ public static Translation3d getNormalTranslation(Translation3d vec){
         return(new Translation3d(vec.getX()/vec.getNorm(), vec.getY()/vec.getNorm(), vec.getZ()/vec.getNorm()));
     }
 
-    // This class is used to store PID Constants
-    public static class PIDParameters{
-        private double mKP;
-        private double mKI;
-        private double mKD;
-        private double mKV;
-        private double mKS;
-        private double mContinuousInputMax;
-        private double mContinuousInputMin;
-        private boolean mIsContinuousInput;
-
-        public PIDParameters(double kP, double kI, double kD, double kS, double kV, double continuousInputMax, double continuousInputMin){
+    public enum TalonControlType {
+        VELOCITY_VOLTAGE,
+        COAST_OUT,
+        VOLTAGE_OUT,
+        DUTY_CYCLE_VEL;}
+
+    public static class PIDConstants{
+        //These are public for the sake of editing during PID tuning (I'm not gonna make 16 getters/setters)
+        public double mKP;
```

Comment:
Public non-final fields should be avoided.

Reply:
Done